README.md

```diff
@@ -121,11 +121,11 @@ docker run --name=redmine -it --rm -p 10080:80 \
 sameersbn/redmine:3.0.2
 ```
 
-Or you can use [fig](http://www.fig.sh/). Assuming you have fig installed,
+Or you can use docker-compose. It's like [fig](http://www.fig.sh/) but it is integrated in docker since docker version 1.2.
 
```

Comment:
@DanielBadura what do you mean it is integrated in docker? Isn't it a separate binary that has to be installed separately in addition to docker?